# Reason: allow oe-lite to do the stripping - otherwise the -dbg packages end up empty
# Source: oe-lite
# Upstream: Probably inappropriate/unacceptable

diff -u -r mosquitto-1.4.10/client/Makefile mosquitto-1.4.10.new/client/Makefile
--- mosquitto-1.4.10/client/Makefile	2016-08-16 23:56:28.000000000 +0200
+++ mosquitto-1.4.10.new/client/Makefile	2016-08-22 13:10:00.881797768 +0200
@@ -24,8 +24,8 @@
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/bin
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_pub ${DESTDIR}${prefix}/bin/mosquitto_pub
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_sub ${DESTDIR}${prefix}/bin/mosquitto_sub
+	$(INSTALL)  mosquitto_pub ${DESTDIR}${prefix}/bin/mosquitto_pub
+	$(INSTALL)  mosquitto_sub ${DESTDIR}${prefix}/bin/mosquitto_sub
 
 uninstall :
 	-rm -f ${DESTDIR}${prefix}/bin/mosquitto_pub
diff -u -r mosquitto-1.4.10/lib/cpp/Makefile mosquitto-1.4.10.new/lib/cpp/Makefile
--- mosquitto-1.4.10/lib/cpp/Makefile	2016-08-16 23:56:28.000000000 +0200
+++ mosquitto-1.4.10.new/lib/cpp/Makefile	2016-08-22 13:10:00.885797781 +0200
@@ -10,7 +10,7 @@
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/lib${LIB_SUFFIX}/
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so.${SOVERSION}
+	$(INSTALL)  libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so.${SOVERSION}
 	ln -sf libmosquittopp.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquittopp.so
 	$(INSTALL) -d ${DESTDIR}${prefix}/include/
 	$(INSTALL) mosquittopp.h ${DESTDIR}${prefix}/include/mosquittopp.h
diff -u -r mosquitto-1.4.10/lib/Makefile mosquitto-1.4.10.new/lib/Makefile
--- mosquitto-1.4.10/lib/Makefile	2016-08-16 23:56:28.000000000 +0200
+++ mosquitto-1.4.10.new/lib/Makefile	2016-08-22 13:10:00.885797781 +0200
@@ -25,7 +25,7 @@
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/lib${LIB_SUFFIX}/
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so.${SOVERSION}
+	$(INSTALL)  libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so.${SOVERSION}
 	ln -sf libmosquitto.so.${SOVERSION} ${DESTDIR}${prefix}/lib${LIB_SUFFIX}/libmosquitto.so
 	$(INSTALL) -d ${DESTDIR}${prefix}/include/
 	$(INSTALL) mosquitto.h ${DESTDIR}${prefix}/include/mosquitto.h
diff -u -r mosquitto-1.4.10/src/Makefile mosquitto-1.4.10.new/src/Makefile
--- mosquitto-1.4.10/src/Makefile	2016-08-16 23:56:28.000000000 +0200
+++ mosquitto-1.4.10.new/src/Makefile	2016-08-22 13:10:00.881797768 +0200
@@ -103,12 +103,12 @@
 
 install : all
 	$(INSTALL) -d ${DESTDIR}$(prefix)/sbin
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto ${DESTDIR}${prefix}/sbin/mosquitto
+	$(INSTALL)  mosquitto ${DESTDIR}${prefix}/sbin/mosquitto
 	$(INSTALL) -d ${DESTDIR}$(prefix)/include
 	$(INSTALL) mosquitto_plugin.h ${DESTDIR}${prefix}/include/mosquitto_plugin.h
 ifeq ($(WITH_TLS),yes)
 	$(INSTALL) -d ${DESTDIR}$(prefix)/bin
-	$(INSTALL) -s --strip-program=${CROSS_COMPILE}${STRIP} mosquitto_passwd ${DESTDIR}${prefix}/bin/mosquitto_passwd
+	$(INSTALL)  mosquitto_passwd ${DESTDIR}${prefix}/bin/mosquitto_passwd
 endif
 
 uninstall :
